src/main/java/taskat3/CaloriesCalculation.java

```diff
@@ -0,0 +1,13 @@
+package taskat3;
+
+import java.util.List;
+
+public class CaloriesCalculation {
+    public static int caloriesCalculation(List<Vegetable> vegetableList) {
+        int calories = 0;
+        for (Vegetable vegetable : vegetableList) {
+            calories += vegetable.getCalories();
+        }
+        return calories;
```

Comment:
this code looks pretty familiar in comparison with code of your colleague, that was pushed earlier than this samples. How is it possible?

Reply:
maybe I was over googling